src/Lecture1.hs

```diff
@@ -1,122 +1,142 @@
-{- |
-Module                  : Lecture1
-Copyright               : (c) 2021-2022 Haskell Beginners 2022 Course
-SPDX-License-Identifier : MPL-2.0
-Maintainer              : Haskell Beginners 2022 Course <haskell.beginners2022@gmail.com>
-Stability               : Stable
-Portability             : Portable
-
-Exercises for the Lecture 1 of the Haskell Beginners course.
-
-To complete exercises, you need to complete implementation and add
-missing top-level type signatures. You can implement any additional
-helper functions. But you can't change the names of the given
-functions.
-
-Comments before each function contain explanations and example of
-arguments and expected returned values.
-
-It's absolutely okay if you feel that your implementations are not
-perfect. You can return to these exercises after future lectures and
-improve your solutions if you see any possible improvements.
--}
-
+-- |
+-- Module                  : Lecture1
```

Comment:
I see that you used the `ormolu` formatter in VSCode.
Because of that, all comments have changed from block to single-line. It's a bit difficult to review as I don't see the diff clearly 😥 

I can review for now but in the future, you can disable `ormolu` in your VSCode settings to keep diffs smaller 🙂